pubspec.yaml

```diff
@@ -8,6 +8,7 @@ dependencies:
   collection: "^1.5.0"
   http_parser: ">=0.0.1 <4.0.0"
   path: ">=0.9.0 <2.0.0"
+  typed_data: "^1.1.5"
```

Comment:
You should generally try to make the lower bound of constraints as low as possible while they still provide the APIs you're using, so that the version solver has more flexibility. In this case, `^1.0.0` should be fine.

Reply:
Just for my own reference are we going to want to tighten the restriction for strong mode compliance?